static/js/settings_org.js

```diff
@@ -244,8 +244,12 @@ function set_message_retention_setting_dropdown() {
 function set_org_join_restrictions_dropdown() {
     const value = get_property_value("realm_org_join_restrictions");
     $("#id_realm_org_join_restrictions").val(value);
-    change_element_block_display_property('allowed_domains_label',
-                                          value === 'only_selected_domain');
+    const zero_domains_warning = $('#zero_domains_warning');
```

Comment:
Can we not make this new warning reuse that show/hide scheme that `change_element_block_display_property` provides?

Reply:
That function hides the block in which `#zero_domains_warning` is present. so, it's hiding the whole table.